src/Compilers/Core/Portable/Generated/Operations.xml.Generated.cs

```diff
@@ -5947,21 +5947,28 @@ internal abstract partial class BaseLocalFunctionStatement : Operation, ILocalFu
         /// Local function symbol.
         /// </summary>
         public IMethodSymbol Symbol { get; }
-        protected abstract IBlockOperation BodyImpl { get; }
+        protected abstract IBlockOperation BlockBodyImpl { get; }
```

Comment:
>BlockBodyImpl [](start = 43, length = 13)

Please get rid of the "Impl" pattern for this node, a merge of this change to dataflow branch shouldn't require any fix-ups, we should be able to simply accept code from the parent.